cortex-python.gocd.yaml

```diff
@@ -51,6 +51,7 @@ pipelines:
                     set -eux
                     git clone git@github.com:CognitiveScale/gocd-pipeline-scripts.git
                     ./gocd-pipeline-scripts/common/c12e-common.sh dev
+                    ./gocd-pipeline-scripts/common/sonar-scan.sh scan
```

Comment:
do we have a way to dfferentiate between branches when this command runs?

Reply:
We cannot differentiate based on branch(as Community edition does nto support branches) but we can differentiate based on commit ID. Sample for PG below
<img width="510" alt="image" src="https://github.com/CognitiveScale/cortex-python/assets/27280368/b1e67949-4b76-4101-a86d-4638ce869588">